rules.d/50-udev-default.rules.in

```diff
@@ -64,6 +64,8 @@ SUBSYSTEM=="sound", GROUP="audio", \
 
 SUBSYSTEM=="usb", ENV{DEVTYPE}=="usb_device", MODE="0664"
 
+SUBSYSTEM=="net", IMPORT{builtin}="net_driver"
```

Comment:
This should be always done except for 'remove' event, i.e. please move this line earlier than
```
ACTION!="add", GOTO="default_end"
```